Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Is the interpretation here correct? I dont use this distribution all that often</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I actually never use these interpretations of probability distributions -- I find them incredibly confusing.</p><p>To me, the NegBin is a Gamma-Poisson, i.e an overdispersed Poisson: a Poisson random variable whose rate parameter is gamma distributed. That means I parametrize it with <code>mu</code>  (Poisson rate) and <code>alpha</code>  (Gamma rate)</p>

Reply:
<p>I get what youre saying but I have a question.</p><p> </p><p>How would reword your explanation for a beginner student that doesn't know what the words Poisson, rate parameter, Gamma, or overdispersed mean?</p><p> </p>

Reply:
<p>Actually I think we should add the Poisson and Gamma in there -- very common and will ease the introduction of NegBin (i.e Gamma-Poisson) if you want to go with my interpretation (which you don't have to; I'm just sharing my own learning path).</p><p> </p><p>Poisson is often used to model the number of independ events occurring in a fixed period of time -- number of goals scored in a football game, number of minutes I'm gonna wait for the bus at my stop, etc. This number is called the Poisson rate and has to be positive.</p><p>Another way to view the Poisson is the number of successes, like Binomial, but without an obvious upper limit.</p><p> </p><p>Gamma is only defined for positive real numbers -- like the HalfNormal but more flexible, or like the Beta but not restricted to [0, 1].</p><p> </p><p>That means you can draw positive numbers from a common Gamma and use each this draw to define a new Poisson, each of which you draw samples from -- that's the NegBin (or Gamma-Poisson)</p><p> </p><p>I like that explanantion because it's just sampling and set you up for the reasoning of hierarchical models. But it's because I just can't relate to examples like "Number of failures until N number of successes" -- I find these super confusing 😅</p>

Reply:
<p>What if used an example thats like number of goals attempts needed to make 2 goals</p>

Reply:
I like that better, although it's still convoluted (why 2 goals? 😉 )

Reply:
<p>Well cause its an example, It could be 3 or 4. Maybe I'll say until N successful goals or something</p><p> </p>